internal/controllers/s3userclaim/handler.go

```diff
@@ -129,3 +129,21 @@ func (r *Reconciler) initVars(req ctrl.Request) {
 
 	r.s3UserName = fmt.Sprintf("%s.%s", req.Namespace, req.Name)
 }
+
+func cephSubUserFullIdMaker(cephUserFullId string, subUser string) string {
+	return fmt.Sprintf("%s:%s", cephUserFullId, subUser)
+}
+
+func subUserSecretNameMaker(s3UserClaimName string, subUser string) string {
+	return fmt.Sprintf("%s-%s", s3UserClaimName, subUser)
+}
+
+func subUserNameExtractor(cephSubUserFullId string) (string, error) {
```

Comment:
In Golang, the convention is to use "agent nouns" (e.g. Extractor) for interfaces, not function names. For functions, we generally use "verbs". For example, we could use here:
`generateSubUserFullId`, `generateSubUserSecretName`, `extractSubUserName`

Reply:
Nice! Thanks.